cmd/kubelet/app/options/options.go

```diff
@@ -309,10 +304,7 @@ func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
 func (f *KubeletFlags) AddFlags(fs *pflag.FlagSet) {
 	f.ContainerRuntimeOptions.AddFlags(fs)
 
-	fs.Var(&f.KubeConfig, "kubeconfig", "Path to a kubeconfig file, specifying how to connect to the API server.")
```

Comment:
leaving `--require-kubeconfig` bound to a local string and inert would let this PR merge (since it wouldn't matter whether kops passed the flag or not). I'd be ok with leaving it that way for a release (would let 1.10 kubelets come up even if people specified the deprecated flag). update the deprecated message to say the flag doesn't do anything any more and will be removed in the future

Reply:
OK， I will add it back soon. Thanks =)